sys/include/net/gnrc/netapi.h

```diff
@@ -103,6 +103,44 @@ typedef struct {
     uint16_t data_len;          /**< size of the data / the buffer */
 } gnrc_netapi_opt_t;
 
+/**
+ * @brief   Shortcut function for sending @ref GNRC_NETAPI_MSG_TYPE_SND or
+ *          @ref GNRC_NETAPI_MSG_TYPE_RCV messages
+ *
+ * @param[in] pid       PID of the targeted network module
+ * @param[in] pkt       pointer into the packet buffer holding the data to send
+ * @param[in] type      type of the message to send. Must be either
+ *                      @ref GNRC_NETAPI_MSG_TYPE_SND or
+ *                      @ref GNRC_NETAPI_MSG_TYPE_RCV
+ *
+ * @return              1 if packet was successfully delivered
+ * @return              -1 on error (invalid PID or no space in queue)
+ */
+int _gnrc_netapi_snd_rcv(kernel_pid_t pid, gnrc_pktsnip_t *pkt, uint16_t type);
```

Comment:
Though I like brevity, too ... but can we have the extra 2 vowels and name the function `_gnrc_netapi_send_recv`, please.

Also for the return value: why not `0` for success and anything else (like `1`) for errors, like most C APIs do?

Reply:
okay for the latter, I now see that this would induce an API change because of changed error values ... so leave as is

Reply:
Exactly ;-). Will change the name though as requested.